gulp/util/webpack-config.js

```diff
@@ -56,7 +56,6 @@ const EXTERNALS = {
     "jquery": "$",
     "moment": "moment",
     "react": "React",
-    "react-addons-css-transition-group": "React.addons.CSSTransitionGroup",
```

Comment:
😕 but you got a preview comment! 🏅 
revert it so the site will load and we can investigate?